### ISSUE TITLE
[Wrath Bug Report] [ADDON_ACTION_BLOCKED] AddOn 'ElvUI' tried to call the protected function 'UNKNOWN()'

### ISSUE THREAD

**Issue by slicedk** [tukui-org/ElvUI#1037]
### What is the issue you are having?

BugSack frequently detecting errors coming from ElvUI WOTLK client: "[ADDON_ACTION_BLOCKED] AddOn 'ElvUI' tried to call the protected function 'UNKNOWN()'"

### What is the expected behavior?

No frequent errors during regular gameplay.

### What actually happened?

BugSack detecting an error coming from ElvUI. Issue persisted after full reinstall of WoW client and ElvUI. Happens also with /edebug on and only BugGrabber/BugSack enabled.

![image](https://github.com/tukui-org/ElvUI/assets/143911024/11a51a87-b434-450f-a86e-00ec6ffa38e0)

### Suggested Solution/Workaround

Probably connected to the quest log/Objective frame.

### Errors

```shell
1x [ADDON_ACTION_BLOCKED] AddOn 'ElvUI' tried to call the protected function 'UNKNOWN()'.
[string "@!BugGrabber/BugGrabber.lua"]:481: in function <!BugGrabber/BugGrabber.lua:481>
[string "=[C]"]: ?
[string "=[C]"]: in function `SetPoint'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:2699: in function `UIParentManageFramePositions'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:1958: in function <Interface_Wrath/FrameXML/UIParent.lua:1945>
[string "=[C]"]: in function `SetAttribute'
[string "@Interface_Wrath/FrameXML/UIParent.lua"]:2714: in function <Interface_Wrath/FrameXML/UIParent.lua:2712>
[string "=[C]"]: in function `UIParent_ManageFramePositions'
[string "@Blizzard_UIWidgets/Blizzard_UIWidgetBelowMinimapFrame.lua"]:25: in function `layoutFunc'
[string "@Blizzard_UIWidgets/Blizzard_UIWidgetManager.lua"]:225: in function `UpdateWidgetSetContainerLayout'
[string "@Blizzard_UIWidgets/Blizzard_UIWidgetManager.lua"]:318: in function `ProcessWidgetSet'
[string "@Blizzard_UIWidgets/Blizzard_UIWidgetManager.lua"]:340: in function `UpdateAllWidgets'
[string "@Blizzard_UIWidgets/Blizzard_UIWidgetManager.lua"]:33: in function <...dOns/Blizzard_UIWidgets/Blizzard_UIWidgetManager.lua:31>

Locals:
_ = Frame {
 RegisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:487
 0 = <userdata>
 UnregisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:487
 SetScript = <function> defined @!BugGrabber/BugGrabber.lua:487
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
 ADDON_ACTION_BLOCKED = <function> defined @!BugGrabber/BugGrabber.lua:553
 ADDON_ACTION_FORBIDDEN = <function> defined @!BugGrabber/BugGrabber.lua:553
 PLAYER_LOGIN = <function> defined @!BugGrabber/BugGrabber.lua:547
 LUA_WARNING = <function> defined @!BugGrabber/BugGrabber.lua:562
 ADDON_LOADED = <function> defined @!BugGrabber/BugGrabber.lua:507
}

```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md).
- [X] I have verified ElvUI is up to date by checking /estatus.
- [X] I have verified my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verified my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by slicedk** [comment]
If I disable "Quest Choice" and "Quest Frames" under "Skins", the error changes as follows:

1x Interface_Wrath/FrameXML/WatchFrame.lua:352: attempt to perform arithmetic on a nil value
[string "@Interface_Wrath/FrameXML/WatchFrame.lua"]:352: in function `WatchFrame_Update'
[string "@Interface_Wrath/FrameXML/QuestLogFrame.lua"]:720: in function `_QuestLog_ToggleQuestWatch'
[string "@Interface_Wrath/FrameXML/QuestLogFrame.lua"]:101: in function <Interface_Wrath/FrameXML/QuestLogFrame.lua:86>

Locals:
self = WatchFrame {
 0 = <userdata>
 linePool = <table> {
 }
 updateTimer = 1
 updating = true
 holder = ObjectiveFrameHolder {
 }
 buttonPool = <table> {
 }
 AutoHider = Frame {
 }
 nextOffset = 0
 SetPoint = <function> defined =[C]:-1
 watchMoney = false
}
pixelsUsed = 0
totalOffset = 0
lineFrame = WatchFrameLines {
 0 = <userdata>
 updateFunctions = <table> {
 }
 numFunctions = 0
}
(*temporary) = nil
(*temporary) = nil
(*temporary) = WatchFrame {
 0 = <userdata>
 linePool = <table> {
 }
 updateTimer = 1
 updating = true
 holder = ObjectiveFrameHolder {
 }
 buttonPool = <table> {
 }
 AutoHider = Frame {
 }
 nextOffset = 0
 SetPoint = <function> defined =[C]:-1
 watchMoney = false
}
(*temporary) = <userdata>
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to perform arithmetic on a nil value"
WatchFrame_ResetLinkButtons = <function> defined @Interface_Wrath/FrameXML/WatchFrame.lua:71
WatchFrame_ReleaseUnusedLinkButtons = <function> defined @Interface_Wrath/FrameXML/WatchFrame.lua:75

**Comment by kodewdle** [comment]
fixed in 13.40

**Comment by slicedk** [comment]
> fixed in 13.40

amazing, thanks
